spec/acceptance/spiped_spec.rb

```diff
@@ -41,10 +41,15 @@
 
   describe 'redis over tunnel' do
     if fact('hostname') == 'spipedserver'
+      redis_service = if fact('operatingsystemmajrelease') == '8'
+                        'redis-server'
+                      else
+                        'redis'
+                      end
       describe command('sed -i "s/^bind.*/bind 127.0.0.1/g" /etc/redis/redis.conf') do
         its(:exit_status) { is_expected.to eq 0 }
       end
-      describe command('systemctl start redis') do
+      describe command("systemctl start #{redis_service}") do
```

Comment:
Should this be `describe service(redis_service) { it { is_expected.to be_running } }` to ensure it's running?